shelldio.sh

```diff
@@ -161,6 +189,7 @@ while [ "$1" != "" ]; do
 		;;
 	-a | --add)
 		welcome_screen
+		validate_station_lists
```

Comment:
```suggestion
		validate_station_lists
		add_stations
		validate_station_lists # θα την ξανάτρεχα εδώ, just to be on the safe side
```

Πέρα από την `add`, λες να ταίριαζε να μπει και πριν κάνει `exit 0` η `-f | --fresh)`;

Reply:
Αν έρχεται λαθος csv απο το repo μετα απο `--fresh` έχουμε σίγουρα πρόβλημα! :D